README.md

```diff
@@ -6,45 +6,69 @@ A Floating UI wrapper for Svelte.
 
 This project is maintained by [Hugo Korte](https://github.com/Hugos68), [Skeleton Labs](https://www.skeletonlabs.co/), and the [Svelte](https://svelte.dev/) community. Based on the amazing work by [Floating UI](https://github.com/floating-ui/floating-ui).
 
-## Contributing
+## Installation
 
-### Developing
-
-Once you've created a project and installed dependencies with `npm install` (or `pnpm install` or `yarn`), start a development server:
+Install the package using your preferred package manager.
 
 ```bash
-npm run dev
-
-# or start the server and open the app in a new browser tab
-npm run dev -- --open
+npm add @skeletonlabs/floating-ui-svelte
```

Comment:
Nitpicking, but although `add` works, `npm install` is the common way when using npm and I think it will be more common to users (`add`, `i` and some others are aliases of `npm install`).

Reply:
Good point, thanks!